var/spack/repos/builtin/packages/texlive/package.py

```diff
@@ -29,11 +29,17 @@ class Texlive(AutotoolsPackage):
     # connection at install time and the package versions could change over
     # time. It is better to use a version built from tarballs, as defined with
     # the "releases" below.
-    version('live', sha256='44aa41b5783e345b7021387f19ac9637ff1ce5406a59754230c666642dfe7750',
-            url='ftp://tug.org/historic/systems/texlive/2019/install-tl-unx.tar.gz')
+    version('live', sha256='7c90a50e55533d57170cbc7c0370a010019946eb18570282948e1af6f809382d',
+            url='ftp://tug.org/historic/systems/texlive/2020/install-tl-unx.tar.gz')
```

Comment:
Is the 2019 version still present? I wonder if we should keep that and make it the default so that the checksum doesn't keep changing on the default version.

Reply:
Yeah it still exists. I'm not sure what to do about it. The note here for texlive considers the `live` version deprecated. I was able to build with the source but I haven't tested it yet. In the past I seem to remember them removing the previous versions so we could only have the single `live` version and it would break when they updated it. We should probably version the `live` versions as well so they are more robust in the future?

Reply:
I vote for removing the `live` version.

Reply:
When I initially had problems building after noticing it switched to source, I was really tempted to just install the `live` version since it always worked fine for me in the past so I think it would be good to keep around.

Reply:
The default version is the numbered version btw. Currently it's `20190410` and with this PR it will be `20200406`. The `live` version has the lowest precedence. I would say I think it makes the most sense to keep the `live` version as the latest. Or since the 2019 version still exists, I would be fine not updating the live version too.